site/docs/resources/faq.md

```diff
@@ -26,6 +27,42 @@ No, it's not.
    As always, TypeScript will auto-complete the parameter names for you.
 3. Perhaps double-check the method signature for [actions](../guide/context#available-actions) on `ctx` [here](https://deno.land/x/grammy/mod.ts?s=Context#Methods), or for API methods (`ctx.api`, `bot.api`) [here](https://deno.land/x/grammy/mod.ts?s=Api#Methods).
 
+## How Can I Access the Chat History?
+
+You can't.
+
+Telegram does not store the messages for your bot.
+There is no way to obtain the data, because it is not there.
+
```

Comment:
Need to remove this too if we accepted my suggestion, and maybe rephrase the sentences after that. Only if we accepted my suggestion tho.